### PR TITLE
Fix to #3991 – Geolocation snaps pin to matched address…

### LINKED ISSUE #3991: Geolocation snaps pin to matched address, unable to set specific coordinates
It no longer appears possible to drag the Geolocation map marker to a location without having it automatically "snap" to the center of the closest Matched Address. It is no longer possible to manually set the coordinates to a non-addressed location (as it was in earlier versions).

Behavior noted in 2.2.5 and 2.2.8.

### PR DESCRIPTION
**Pull request by rarila**
Fixes #3991 
@bobdenotter Here to test: Use it that way or remove the snap checkbox and go with what option?

### CODE REVIEW

**Comment by bobdenotter** [comment]
I'm not necessarily a big fan of this, but if people want to have this, who am I to block it? :-)

It seems to work fine, so merging in!

**Comment by rossriley** [comment]
You reckon we could have this ported to 2.2 branch too?

**Comment by bobdenotter** [comment]
> You reckon we could have this ported to 2.2 branch too?

I have no strong feelings either way. ^_^

File: app/view/twig/editcontent/fields/_geolocation.twig [hunk 1]
```diff
@@ -41,7 +41,14 @@
...
         type:        'text',
-    }
+    },
+
+    snap: {
+        checked:     true,
+        class:       'snap',
+        name:        name ~ '_snap',
+        type:        'checkbox',
+    },
```
Comment (bobdenotter) [inline app/view/twig/editcontent/fields/_geolocation.twig]: This comma shouldn't be here.

**Comment by bobdenotter** [comment]
@rossriley See #4093

**Pull request merged**